test/api/buffer/exports/delete/006-get-ok/expected-http-code

```diff
@@ -0,0 +1 @@
+200
```

Comment:
Možná bys teda mohl doplnit ještě 404ku na GET toho smazanýho, ať je to komplet.

Reply:
Na export samotný nemůžeš dát `GET`, nemáme na to endpoint. `005-delete-404` testuje, že pak ten stejný export už nejde smazat, protože neexistuje. Potom v `get-ok` je vidět, že receiver už ten export taky nemá. Stačí to tak?

Reply:
Mali by sme myslim pridat get na export, ked mame PATCH, nejak sa na to asi iba zabudlo.

Budu tam aj nejake statistiky, tak sa to moze hodit, nacitat iba jeden export.

Reply:
Ahá jasně. 🙈 Ok.